src/constants/extensions.ts

```diff
@@ -9,6 +9,15 @@ import {
   convertPeriodToSeconds,
 } from '~utils/extensions';
 
+import oneTransactionHero from '~images/assets/extensions/one-transaction-hero.png';
+import oneTransactionInterface from '~images/assets/extensions/one-transaction-interface.png';
+import reputationHero from '~images/assets/extensions/reputation-hero.png';
+import reputationInterface from '~images/assets/extensions/reputation-interface.png';
+import advancedHero from '~images/assets/extensions/advanced-hero.png';
+import advancedInterface from '~images/assets/extensions/advanced-interface.png';
+import streamingHero from '~images/assets/extensions/streaming-hero.png';
+import streamingInterface from '~images/assets/extensions/streaming-interface.png';
```

Comment:
I'm not sure about the `assets` part of the path, looking at the current structure of the images folder putting `extensions` at the root seems more consistent